omod/src/main/java/org/openmrs/module/idgen/rest/resource/LogEntryResource.java

```diff
@@ -0,0 +1,90 @@
+package org.openmrs.module.idgen.rest.resource;
```

Comment:
You will also need to add the openmrs license header file.

Reply:
Fixed this. Should reflect on the next commit :)